documentation/general/quickstart.md

```diff
@@ -6,6 +6,63 @@ This guide covers the steps to get the Openverse stack running locally on your
 computer. This guide is for setting up the full stack, which includes the API,
 the ingestion server and the frontend.
 
+## Using the Openverse development environment
+
+```{tip}
+The `ov` script is only intended for Linux and macOS development environments.
+
+Windows users must use WSL for Openverse development, even when using the `ov`
+script.
+```
+
+```{warning}
+This approach is experimental and still under development! It might not work
+for all situations related to Openverse development yet, but we hope it will soon.
+If you try this out and run into any problems, please open an issue on GitHub
+so we can fix it.
+```
+
+The `ov` script at the root of the Openverse repository is capable of managing
+all dependencies needed to work on Openverse, provided the following are
+available on your computer:
+
+- git
+- bash
+- Docker (just the container runtime)
+
+```{note}
+macOS and most entry-level Linux distributions come with git and bash installed.
+This means Docker is the only dependency most Openverse contributors need to
+install on their base system when using the `ov` script.
+
+Follow Docker's official installation instructions if you don't already have it:
+
+https://docs.docker.com/engine/install/
+
+Contributors should consult their system manual to install `git` and `bash` if
+it is not available by default.
+```
+
+With just these three dependencies available on your computer, you can use the
+`ov` script to run the entire Openverse development environment without worrying
+about other more specific dependencies.
+
+[Follow these instructions to clone the repository](#clone-the-repository). Then
+run the `ov init` script from the repository's root directory:
+
+```bash
+ov init
```

Comment:
Calling `ov` needs the `./` prefix?

```suggestion
./ov init?
```

<img width="757" alt="Screenshot 2024-05-28 at 1 23 49 PM" src="https://github.com/WordPress/openverse/assets/16580576/13afd594-b244-4011-8edf-e1e7d5d6809f">

Reply:
I missed the step to `ov link`. Sorry.